packages/core/components/AutoField/index.tsx

```diff
@@ -102,7 +102,10 @@ export type InputProps<F = Field<any>> = {
   readOnlyFields?: Record<string, boolean | undefined>;
 };
 
-export const InputOrGroup = ({ onChange, ...props }: InputProps) => {
+export function AutoField<FieldType extends Field>({
```

Comment:
Change to `function` here deliberate?

Reply:
Yes, to support generic types! Newer TS compilers do actually support generic types for arrow functions now, but I still do this by force of habit